### PR TITLE
LVM-activate: replace error log messages with calls to ocf_exit_reason

### PR DESCRIPTION
**Pull request by XingWei-Liu**
issue #704  
LVM-activate RA update ocf_exit_reason support

### CODE REVIEW

**Comment by oalbrigt** [comment]
Can you undo the changes for the couple of places where it does logging before setting exit reason?

**Comment by XingWei-Liu** [comment]
Thank you for your reminder, I have completed the modification, is there anything else that needs to be modified?

File: heartbeat/LVM-activate [hunk 1]
```diff
@@ -441,7 +441,7 @@ tagging_check()
 	# The volume_list must be initialized to something in order to
 	# guarantee our tag will be filtered on startup
 	if ! lvm dumpconfig activation/volume_list; then
-		ocf_log err  "LVM: Improper setup detected"
+		ocf_log_err  "LVM: Improper setup detected"
```
Comment (oalbrigt) [inline heartbeat/LVM-activate:444]: These needs to be updated to `ocf_log err`.

Comment (XingWei-Liu) [reply heartbeat/LVM-activate]: Sorry for my carelessness , I have completed the modification

Comment (oalbrigt) [reply heartbeat/LVM-activate]: No worries.

**Comment by oalbrigt** [comment]
Thanks.

**Pull request merged**